### PR TITLE
[Schema] Fix publish schema

### PR DESCRIPTION
**Pull request by Yashsharma1911**
**Description**

This PR fixes #
Removed comma after Kubernetes in technology enum, which was causing error

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by Yashsharma1911** [comment]
Merging this PR as this is small change

**Comment by leecalcote** [comment]
Thanks for this, @Yashsharma1911 👍

File: schemas/configuration/publishCatalogItem.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
             "title": "Technology",
             "items": {
                 "enum": [
-                    "kubernetes",
+                    "kubernetes"
```
Comment (MUzairS15) [inline schemas/configuration/publishCatalogItem.json:10]: RJSF crashed due to this, instead RJSF should have thrown validation issue IMO.

Comment (MUzairS15) [reply schemas/configuration/publishCatalogItem.json:10]: Do we have that control?

**Pull request merged**